web/src/filter.ts

```diff
@@ -884,6 +885,10 @@ export class Filter {
     }
 
     sorted_terms_for_comparison(excluded_operators?: string[]): string[] {
+        if (this.cached_sorted_terms_for_comparison !== undefined) {
+            return this.cached_sorted_terms_for_comparison;
+        }
```

Comment:
Also this ignores the `excluded_operators` parameter.

Reply:
Fixed to not cache or use cached value if excluded_operators is passed.